src/screens/DomainScreen/elements/Header/index.js

```diff
@@ -153,7 +153,7 @@ const styles = StyleSheet.create({
   },
   followButtonText: {
     fontSize: normalizeFontSize(14),
-    color: COLORS.white,
+    color: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the `color` property in the `followButtonText` style might affect readability if the background color of this text is not adjusted accordingly. Please ensure that the background color contrasts well with this new text color for better accessibility and readability.
```diff
-    color: COLORS.white,
+    color: COLORS.almostBlack,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->